### PR TITLE
Add benchmarks

### PR DESCRIPTION
**Pull request by ehmicky**
This adds some benchmarks.

Some findings:
  - `get-stream` is roughly as fast as core Node.js
  - `getStream()` is faster than `getStreamAsBuffer()`/`getStreamAsArrayBuffer()` on string-based Node.js streams, slower otherwise
  - `getStreamAsArray()` is always faster than the other methods
  - Web streams with a "bytes controller" are slower than Node.js-specific streams with `getStream()`
  - Web streams with a "default controller" are faster than Node.js-specific streams when the output is binary, but slower when the output is text

### CODE REVIEW

File: benchmarks/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import getStream, {getStreamAsBuffer, getStreamAsArrayBuffer, getStreamAsArray} from '../index.js';
+import {createFixture, deleteFixture, FIXTURE_HUMAN_SIZE} from './fixture.js';
+import {measureTask} from './measure.js';
+import {createNodeStreamBinary, createNodeStreamText, createWebStreamBinary, createWebStreamText} from './stream.js';
+
+await createFixture();
+
+try {
+	console.log(`### Node.js stream (${FIXTURE_HUMAN_SIZE}, binary)\n`);
+	console.log(`- getStream(): ${await measureTask(createNodeStreamBinary, getStream)}ms`);
```
Comment (sindresorhus) [inline benchmarks/index.js:11]:
```suggestion
	console.log(`- \`getStream()\`: ${await measureTask(createNodeStreamBinary, getStream)}ms`);
```

File: benchmarks/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import {text, buffer, arrayBuffer} from 'node:stream/consumers';
+import getStream, {getStreamAsBuffer, getStreamAsArrayBuffer, getStreamAsArray} from '../index.js';
+import {createFixture, deleteFixture, FIXTURE_HUMAN_SIZE} from './fixture.js';
+import {measureTask} from './measure.js';
+import {createNodeStreamBinary, createNodeStreamText, createWebStreamBinary, createWebStreamText} from './stream.js';
+
+await createFixture();
+
```
Comment (sindresorhus) [inline benchmarks/index.js:8]: Maybe use a helper method? Something like this.

```js
const log = (name, promise) => {
  console.log(`- \`${name}()\`: ${await promise}ms`);
};

File: readme.md [hunk 3]
```diff
@@ -171,6 +172,52 @@ const {body: readableStream} = await fetch('https://example.com');
 console.log(await getStream(readableStream.pipeThrough(textDecoderStream)));
 ```
 
+## Benchmarks
+
+### Node.js stream (100MB, binary)
```
Comment (sindresorhus) [inline readme.md:177]:
```suggestion
### Node.js stream (100 MB, binary)
```

File: benchmarks/index.js (hunk 1 above)
Comment (ehmicky) [reply benchmarks/index.js]: Fixed in be99fad

File: benchmarks/index.js (hunk 2 above)
Comment (ehmicky) [reply benchmarks/index.js:8]: Thanks, this is fixed in 8f0ebe7. This looks much better now, my original code was quite messy. :)

File: readme.md (hunk 3 above)
Comment (ehmicky) [reply readme.md]: Done in 825438d.

**Comment by ehmicky** [comment]
Fixed :+1:

**Pull request merged**